src/WingetCreatePackage/Package.appxmanifest

```diff
@@ -21,6 +21,7 @@
 
   <Dependencies>
     <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.18362.0" />
+    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.25426.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
```

Comment:
This will fix the issue when the app is installed from an msix, but not when run as a standalone (like in a CI/CD pipeline, etc)

We should add something to the readme I guess that mentions that this package is required

Reply:
Would be good to add that info to the issue too

Reply:
I added information about the package dependencies to the README.md documentation. I also created a GitHub issue to track the progress of improving the standalone exe to include the necessary binaries of the framework package here: #145